src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx

```diff
@@ -40,7 +40,7 @@ Flex comes bundled with the New Relic infrastructure agent. To create a flex con
   /etc/newrelic-infra/integrations.d
   ```
 
-2. Use the below snippet to update your configuration file named `sendmail-flex-config.yml`.
+2. Update your configuration file `sendmail-flex-config.yml` using the following snippet:
```

Comment:
@pkosanam Your original version said "Below are the fields which are to be updated on the below snippet." 

Which fields are they supposed to update?

Reply:
hi @ally-sassman , user has to update the value "FAILED_MESSAGES_FILE_NAME" present in the snippet, else there will be error recorded on integration.